WebRoot/js/zimbraMail/mail/view/prefs/ZmMailPrefsPage.js

```diff
@@ -834,40 +859,36 @@ function() {
 	this._numUsedText.innerHTML = AjxMessageFormat.format(ZmMsg.whiteBlackNumUsed, [this._listView.size(), this._max]);
 };
 
+ZmWhiteBlackList.prototype._SelectionListener = function (ev) {
+	var selection = this._listView.getSelection();
+	if (ev && ev.target && this._listView.delButtons[ev.target.id]) {
```

Comment:
Not sure, but shouldn't this be `onClick` callback for delete button that we are adding above?